packages/web-components/src/components/va-table/va-table.tsx

```diff
@@ -66,6 +103,29 @@ export class VaTable {
     });
   }
 
+  componentDidUpdate() {
+    // Update the sort icon
+    if (this.sortColumn >= 0) {
+      const icon = this.sortAscending ? descendingIcon : ascendingIcon;
+      const button = this.el.querySelector('va-table-row[slot] button');
+      const node = button.children[0];
+      node.outerHTML = icon;
+    }
+  }
+
+  private handleSort(): void {
+    const cellSelector = row => row.children[this.sortColumn].textContent;
+    // Starting at index 1 to skip the header row
+    const sortingFunc = this.sortAscending ? quicksort : reverseQuicksort;
+    sortingFunc(
+      this.el.childNodes,
+      1,
+      this.el.childNodes.length - 1,
+      cellSelector,
+    );
```

Comment:
Here we call quicksort with a live `NodeList`. It happens in place because I didn't want to have to develop our own virtual DOM or other DOM abstraction.